ironfish-cli/src/commands/wallet/mint.ts

```diff
@@ -60,6 +60,11 @@ export class Mint extends IronfishCommand {
         'Minimum number of block confirmations needed to include a note. Set to 0 to include all blocks.',
       required: false,
     }),
+    rawTransaction: Flags.boolean({
+      default: false,
+      description:
+        'Return raw transaction. Use it to create a transaction but not post to the network',
```

Comment:
maybe `Return the raw transaction. Used to create a transaction but not post to the network`